jedi_vim.py

```diff
@@ -417,9 +416,6 @@ def rename():
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
 
-            result = new_buffer(window_path)
-            if not result:
-                return
             vim.current.window.cursor = cursor
```

Comment:
@petobens 
Are you sure it breaks renaming in multiple files?

Multiple files should be handled by the `new_window` for `vim.current.buffer.name != r.module_path` above.

So from a quick glance it looks like these lines should be removed really.

I've started looking deeper into this - PR will follow.